view/common/block-layout/browse-preview.phtml

```diff
@@ -16,13 +16,13 @@ $currentLayout = (!isset($layoutSetting) || strpos($layoutSetting, 'grid') !== f
 <div class="browse-controls">
     <div class="layout-toggle">
         <button type="button" aria-label="<?php echo $translate('Grid'); ?>" class="grid o-icon-grid" <?php echo $gridState; ?>></button>
-        <button type="button" aria-label="<?php echo $translate('List'); ?>" class="list o-icon-list" <?php echo $listState; ?>></button>        
+        <button type="button" aria-label="<?php echo $translate('List'); ?>" class="list o-icon-list" <?php echo $listState; ?>></button>
     </div>
 </div>
 <?php endif; ?>
 
 
-<ul class="resource-list preview <?php echo $currentLayout; ?>">
+<ul class="resource-list preview grid">
```

Comment:
This ignores the theme's browse layout setting, keeping it fixed at grid. Is there something about the layout setting that interferes with the new browse preview controls?

Reply:
not sure why I made that change--I have a memory of maybe fixing it to be in line with another theme but can find no evidence of this. Fixed in new commit.